src/tools/rest-backend/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_package (CppCMS)
+find_package (Boost 1.50 COMPONENTS system filesystem thread REQUIRED)
+
+if (NOT CppCMS_FOUND)
+    message(SEND_ERROR "CppCMS package not found")
```

Comment:
Ah ok, it didn't make sense to me that you wrote "there should be a big fat warning when `remove_*` is used" but at the same time insist on removing the warning. But now I got it and will remove it.

Warnings should be removed (see last commit some days ago). I used different settings when compiling locally (e.g. no -Wshadow).